packs/ansible/actions/command.yaml

```diff
@@ -0,0 +1,108 @@
+---
+name: "command"
+description: "Run ad-hoc ansible command (module)"
+runner_type: "local-shell-script"
+entry_point: "ansible.py"
+enabled: true
+parameters:
+  kwarg_op:
+    description: "Lock operator type to '--'"
+    type: string
+    default: "--"
+    immutable: true
+  sudo:
+    description: "Lock sudo, the behavior is controlled by ansible 'become_' options"
+    type: boolean
+    default: true
+    immutable: true
+  cwd:
+    description: "Working directory where the command will be executed in"
+    type: string
+  timeout:
+    description: "Action timeout in seconds. Action will get killed if it doesn't finish in timeout seconds"
+    type: integer
+    default: 600
+  hosts:
+    description: "Hosts"
+    type: string
+    position: 0
+  args:
```

Comment:
No,
there is also `list-hosts`: 
`st2 run ansible.command hosts=all list-hosts=true`

And I didn't removed simple "standalone" parameters like `version` and `help` which could be useful during chatops sessions, for example to get quickly ansible version.